adr/0013-home-assistant-container.md

```diff
@@ -0,0 +1,71 @@
+# 0012. Installation method: Home Assistant Container
+
+Date: 2020-06-03
+
+## Status
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+This is for running just the Home Assistant Core application on native OCI compatible containerization system. It does not provide the full Supervisor experience, and thus does not provide the Supervisor panel and add-ons.
+
+This is a general installation method that is recommended as an alternative to the Home Assistant One installation method. Due to the shared image with the Home Assistant One installation method, almost all documentation applies to the Home Assistant Container as well.
+
+The only supported way to run the container is on the host network as root with full privileges.
+
+### Supported Containerization system and version
+
+- Any Open Container Initiative (OCI) compatible containerization system.
+
+### Supported boards/hardware/machines
+
+- Machines of the following architectures: amd64, i386, armhf, aarch64, armv7
+
+### Supported Operating Systems and versions
+
+- Running Home Assistant Container, is only supported on Linux.
+- Windows, and BSD installations (e.g., macOS and FreeBSD) are not supported.
+
+### Additional notes
+
+There is a wide variety of containerization software available. From that perspective, Home Assistant will only actively document the use of Docker.
+
+### Required Expertise
+
+- **Installation**
+  This requires the user to have an existing system that can run Docker containers. Installation is either done by running a command from the Docker-cli or via a user interface (Synology, Portainer)
+
+* **Start when the system is started:** up to the container manager that the user uses.
+* **Run with full network access:** Default installation instructions prescribe net=host but the user can diverge. Usually happens when the user decides it’s not the Docker-way.
+* **Access USB devices:** Responsibility of user.
+
+* **Maintenance Home Assistant installation**
+  If using the Docker-cli user needs to manually update the run command. If using a UI the user might be notified of an upgrade and automatically update. There is no rollback in case the instance does not come online after an update.
```

Comment:
> If using the Docker-cli user needs
to
If using the Docker-cli the user needs

> notified of an upgrade and automatically update
Don't we normally advise against auto-updates?
> notified of an upgrade or automatically update - automatically applying updates may result in the system not coming back online.